### PR TITLE
SC-16584: Update of swagger-ui image and Glue Infrastructure preparation

### PR DESCRIPTION
**Pull request by fsmeier**
### Description

- Ticket/Issue: [SC-16584](https://spryker.atlassian.net/browse/SC-16584)

As a developer I want to see my RESTful API schema not only for the Old Glue API but also for the new Glue API infrastructure (Glue Storefront API & Glue Backend API). I want to be able to use the same tooling but select different schemas based on my environment configuration.

#### Change log

- Update of swagger-ui to v5.9.0
- Integration of new Glue Infrastructure to swagger-ui

### Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: generator/src/templates/service/swagger-ui/v5.9/swagger-ui.yml.twig [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    image: swaggerapi/swagger-ui:v5.9.0
+    networks:
+        - private
+    labels:
+        'spryker.app.name': swagger
+        'spryker.app.type': hidden
+        'spryker.project': ${SPRYKER_DOCKER_PREFIX}:${SPRYKER_DOCKER_TAG}
+    environment:
+        URLS: "[\
+            {% if firstEndpointMap['glue'] is defined  %}{ url: \"{{ protocol ~ firstEndpointMap['glue'] }}/schema.yml\", name: \"(Old) Glue API\" },\{% endif %}
```
Comment (zyuzka) [inline generator/src/templates/service/swagger-ui/v5.9/swagger-ui.yml.twig:13]: Mb better move logic into the index.php file and provide mapped data in template

File: generator/src/templates/service/swagger-ui/v5.9/swagger-ui.yml.twig [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            {% if firstEndpointMap['glue-storefront'] is defined  %}{ url: \"{{ protocol ~ firstEndpointMap['glue-storefront'] }}/schema.yml\", name: \"Glue Storefront API\" },\{% endif %}
+            {% if firstEndpointMap['glue-backend'] is defined  %}{ url: \"{{ protocol ~ firstEndpointMap['glue-backend'] }}/schema.yml\", name: \"Glue Backend API\" },\{% endif %}
+            ]"
+    healthcheck:
+        test: [ "CMD", "nc", "-z", "localhost", "8080" ]
+        interval: 5s
+        timeout: 5s
+        retries: 5
+
```
Comment (zyuzka) [inline generator/src/templates/service/swagger-ui/v5.9/swagger-ui.yml.twig:22]: empty lines

File: generator/src/templates/service/swagger-ui/v5.9/swagger-ui.yml.twig (hunk 1 above)
Comment (zyuzka) [reply generator/src/templates/service/swagger-ui/v5.9/swagger-ui.yml.twig:13]: Is the old Swagger version supported URLS env variable? If yes, we need to adjust both templates

File: generator/src/templates/nginx/http/gateway/glue-storefront.server.conf.twig [hunk 3]
```diff
@@ -1 +1,15 @@
-{% include 'nginx/http/gateway/glue.server.conf.twig' with _context %}
+{% extends "nginx/http/gateway/server.conf.twig" %}
+{% block locations %}
+    {{ parent() }}
+    location /schema.yml {
+    proxy_set_header Host $host;
```
Comment (zyuzka) [inline generator/src/templates/nginx/http/gateway/glue-storefront.server.conf.twig:5]: Move the template to a separate file and include it with $cli_proxy_pass

File: bin/standalone/cli-rpc.js [hunk 4]
```diff
@@ -93,6 +93,76 @@ class Dispatcher {
...
+    }
+
+    _options_glueBackendSchema(request, response) {
+        response.setHeader('Access-Control-Allow-Origin', '*');
+        response.setHeader('Access-Control-Allow-Methods', 'GET');
+        response.statusCode = 201;
+        response.end();
+    }
+
+    _get_glueBackendSchema(request, response) {
```
Comment (zyuzka) [inline bin/standalone/cli-rpc.js:139]: As I see - all Glue* methods are the same. Why do we copy-paste all of them?

**Review by zyuzka — CHANGES_REQUESTED** [review]

**Review by zyuzka — APPROVED** [review]

**Review by alexanderM91 — APPROVED** [review]

**Pull request merged**